gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -950,3 +950,23 @@ verboseHandlerHeapResize(J9HookInterface** hook, uintptr_t eventNum, void* event
 {
 	((MM_VerboseHandlerOutput*)userData)->handleHeapResize(hook, eventNum, eventData);
 }
+
+#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+void
+MM_VerboseHandlerOutput::handleReleaseFreeMemoryPages(J9HookInterface** hook, uintptr_t eventNum, void* eventData)
+{
+	MM_ReleaseFreeMemPagesEvent* event = (MM_ReleaseFreeMemPagesEvent*)eventData;
+	MM_VerboseManager* manager = getManager();
+	MM_VerboseWriterChain* writer = manager->getWriterChain();
+	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	char tagTemplate[200];
+	getTagTemplate(tagTemplate, sizeof(tagTemplate), manager->getIdAndIncrement(), omrtime_current_time_millis());
+	enterAtomicReportingBlock();
+	writer->formatAndOutput(env, 0, "<heap-decommit %s>", tagTemplate);
```

Comment:
**Keyword for new verbose tag:**
<heap-decommit id="4283" timestamp="2017-05-03T04:19:52.563">
  <decommit decommitBytes="411484160" durationms="6.539" />
</heap-decommit>
**>>>to<<<**
<heap-pages id="4283" timestamp="2017-05-03T04:19:52.563">
  <pages releasedBytes="411484160" durationms="6.539" />
</heap-pages>
is it fine?